MediaBrowser.Providers/Plugins/StudiosImages/Configuration/config.html

```diff
@@ -0,0 +1,58 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <title>AudioDB</title>
+</head>
+<body>
+    <div data-role="page" class="page type-interior pluginConfigurationPage configPage" data-require="emby-input,emby-button,emby-checkbox">
+        <div data-role="content">
+            <div class="content-primary">
+                <form class="configForm">
+                    <div class="inputContainer">
+                        <input is="emby-input" type="text" id="repository" required label="Repository" />
+                        <div class="fieldDescription">This can be any Jellyfin-compatible artwork repository.</div>
+                    </div>
+                    <br />
+                    <div>
+                        <button is="emby-button" type="submit" class="raised button-submit block"><span>Save</span></button>
+                    </div>
+                </form>
+            </div>
+        </div>
+        <script type="text/javascript">
+            var PluginConfig = {
+                pluginId: "872a7849-1171-458d-a6fb-3de3d442ad30"
+            };
+
+            document.querySelector('.configPage')
+                .addEventListener('pageshow', function () {
+                    Dashboard.showLoadingMsg();
+                    ApiClient.getPluginConfiguration(PluginConfig.pluginId).then(function (config) {
+                        var repository = document.querySelector('#repository');
+                        repository.value = config.RepositoryUrl;
+                        repository.dispatchEvent(new Event('change', {
+                            bubbles: true,
+                            cancelable: false
+                        }));
+
+                        Dashboard.hideLoadingMsg();
+                    });
+                });
+
+            document.querySelector('.configForm')
+                .addEventListener('submit', function (e) {
+                    Dashboard.showLoadingMsg();
+
+                    ApiClient.getPluginConfiguration(PluginConfig.pluginId).then(function (config) {
```

Comment:
```suggestion
                    window.ApiClient.getPluginConfiguration(PluginConfig.pluginId).then(function (config) {
```

iirc `ApiClient` is now a part of `window`

Reply:
Hm, we might have to update MusicBrainz and AudioDB then, this was taken from there.